app/helpers/roster_helper.rb

```
@@ -112,4 +112,59 @@ def roster_group_badge(group, _group_type)
+                             bs_title: t("registration.item.hints.no_propagation") })
+        safe_join([base_type, " ", icon])
+      end
+    end
+  end
+
+  def format_date(time)
+    return "" if time.nil?
+
+    time.strftime("%b %d, %H:%M")
+  end
+  SELF_ROSTER_TABLE_CONFIG = {
```

RuboCop (1.86.0)

[fatal] dynamic constant assignment
(Using Ruby 2.7 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)